packages/react/src/themes/teams/components/Chat/chatMessageVariables.ts

```diff
@@ -37,8 +39,10 @@ export default (siteVars): ChatMessageVariables => ({
   offset: pxToRem(100),
   padding: pxToRem(16),
   authorMarginRight: pxToRem(12),
-  authorFontWeight: siteVars.fontWeightBold,
+  authorColor: siteVars.gray02,
+  authorFontWeight: siteVars.fontWeightRegular,
```

Comment:
Is this even needed if it's regular?

Reply:
You're right, but we already have this variable and I don't want to remove it because it will be a breaking change...